packages/wallets/component-tests/mocks/mockWalletsAuthorize.ts

```diff
@@ -62,12 +62,7 @@ export const ACCOUNTS_LIST = [
         is_disabled: 0,
         is_virtual: 0,
         landing_company_name: 'svg',
-        linked_to: [
-            {
-                loginid: 'CRW1003',
-                platform: 'dwallet',
-            },
-        ],
+        linked_to: [],
```

Comment:
@fasihali-deriv why are removing this ?
i could'nt see "dwallet" as part of platformContext

Reply:
Hi @amina-deriv as @matin-deriv mentioned in this comment https://github.com/binary-com/deriv-app/pull/14951#discussion_r1590488147 to remove "dwallet" as well. So I removed "dwallet" as well from then project.